src/cascadia/TerminalControl/TermControl.h

```diff
@@ -64,7 +73,7 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         hstring Title();
         hstring GetProfileName() const;
 
-        bool CopySelectionToClipboard(bool collapseText = false);
+        bool CopySelectionToClipboard(bool singleLine = false, Windows::Foundation::IReference<Settings::CopyFormat> formats = Settings::CopyFormat::All);

```

Comment:
*absolutely do not do two default params*

and don't do a default param of a winrt type. those are expensive to construct. we want to make sure we're only constructing them if absolutely necessary.